inlong-sort/sort-connectors/hbase/src/main/java/org/apache/inlong/sort/hbase/sink/HBaseSinkFunction.java

```diff
@@ -161,20 +157,20 @@ public void open(Configuration parameters) throws Exception {
                                     }
                                     try {
                                         flush();
-                                        if (metricData.getNumRecordsOut() != null) {
-                                            metricData.getNumRecordsOut().inc(rowSize);
+                                        if (sinkMetricData.getNumRecordsOut() != null) {
+                                            sinkMetricData.getNumRecordsOut().inc(rowSize);
                                         }
-                                        if (metricData.getNumRecordsOut() != null) {
-                                            metricData.getNumBytesOut()
+                                        if (sinkMetricData.getNumRecordsOut() != null) {
+                                            sinkMetricData.getNumBytesOut()
                                                     .inc(dataSize);
```

Comment:
`sinkMetricData.getNumRecordsOut() != null` to `sinkMetricData.getNumBytesOut() != null`

Reply:
DONE